internal/prometheus_test.go

```diff
@@ -182,9 +182,9 @@ func TestQueryErrorOnQuerying(t *testing.T) {
 }
 
 func TestNormalize(t *testing.T) {
-	s := "1234567890)=azertyuiop^$qsdfghjklm*	<wxcvbn,;:!"
+	s := "1234567890)=azertyuiop^$qsdfghjklm*	<wxcvbn,;:!./-"
```

Comment:
It doesn't concern directly your PR, but could you please add `A` to `Z` to the test input ? This way, everything should be tested.

Reply:
I've added uppercases.

Reply:
Great